src/current/_includes/releases/v24.3/v24.3.0-alpha.1.md

```diff
@@ -1,3 +1,340 @@
+- Added support for configuring authorization using LDAP. During login, the list of groups that a user belongs to are fetched from the LDAP server. These groups are mapped to [SQL roles]({% link v24.3/create-role.md %}) by extracting the common name (CN) from the group. After authenticating the user, the login flow grants these roles to the user, and revokes any other roles that are not returned by the LDAP server. The groups given by the LDAP server are treated as the sole source of truth for role memberships, so any roles that were manually granted to the user will not remain in place. [#131043][#131043]
+- Previously, the HBA configuration cluster setting `server.host_based_authentication.configuration` was unable to handle double quotes in authentication method option values. For example, for the following entry:
+
+    ~~~
+    host all all all ldap ldapserver=ldap.example.com ldapport=636 ldapbasedn="ou=users,dc=example,dc=com" ldapbinddn="cn=readonly,dc=example,dc=com" ldapbindpasswd=readonly_password ldapsearchattribute=uid ldapsearchfilter="(memberof=cn=cockroachdb_users,ou=groups,dc=example,dc=com)"
+    ~~~
+
+    The HBA parser failed after determining `ldapbinddn="cn=readonly,dc=example,dc=com" as 2 separate options(ldapbinddn=and cn=readonly,dc=example,dc=com)`. Now, the 2 tokens can be set as key and value respectively for the same HBA configuration option. [#131480][#131480]
+
+<h3 id="v24-3-0-alpha-1-general-changes">General changes</h3>
+
+- CockroachDB will now avoid [logging]({% link v24.3/logging.md %}) unnecessary stack traces while executing [scheduled jobs]({% link v24.3/show-jobs.md %}). [#129846][#129846]
+- Upgrading to 24.3 is blocked if no license is installed, or if a trial/free license is installed with telemetry disabled. [#130576][#130576]
+- Changed the license `cockroach` is distributed under to the new CockroachDB Software License. [#131661][#131661]
+- Attempting to install a second Enterprise trial license on the same cluster will now fail. [#131422][#131422]
+- Changed the license `cockroach` is distributed under to the new CockroachDB Software License (CSL). [#131690][#131690]
+- Changed the license `cockroach` is distributed under to the new CockroachDB Software License (CSL). [#131686][#131686]
+- Changed the license `cockroach` is distributed under to the new CockroachDB Software License (CSL). [#131688][#131688]
+- Changed the license `cockroach` is distributed under to the new CockroachDB Software License (CSL). [#131687][#131687]
+- Changed the license `cockroach` is distributed under to the new CockroachDB Software License (CSL). [#131717][#131717]
+- Changed the license `cockroach` is distributed under to the new CockroachDB Software License (CSL). [#131689][#131689]
```

Comment:
@rmloveland For each of these, should we update which license it was changed from as per the PR description?

Reply:
OK cool, thanks Rich. It looked like there were a number of existing licenses. Your idea makes sense I will do this!